chain/client/src/client.rs

```diff
@@ -57,6 +54,13 @@ use near_network::types::PartialEncodedChunkForwardMsg;
 
 const NUM_REBROADCAST_BLOCKS: usize = 30;
 
+/// The time we wait for the response to a Epoch Sync request before retrying
+// TODO #3488 set 30_000
+pub const EPOCH_SYNC_REQUEST_TIMEOUT_MS: u64 = 1_000;
```

Comment:
shouldn't they be `Duration`?

Reply:
@bowenwang1996 Yes, I split changing type and moving them out to make it easier to review.